gm4_blast_furnaces/data/blast_furnaces/functions/destroy.mcfunction

```diff
@@ -0,0 +1,6 @@
+#@s - @e[type=armor_stand,tag=gm4_bf_output] unless block ~ ~ ~ BLAST_FURNACE
+#called by processes
+
+loot spawn ~ ~.6 ~ loot blast_furnaces:destroy_blast_furnace
+particle block furnace[lit=true] ~ ~1 ~ .1 .25 .1 .05 30 normal @a
```

Comment:
Breaking a hopper already produces its own particles. Why does this need extra particles, especially from a lit furnace?